Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1120,6 +1121,32 @@ def write(self, msg):
                 self.messages.append(text)
             else:
                 demisto.info(text)
+                if is_debug_mode() and text.startswith('send:'):
```

Comment:
better to have it in a helper function - will make it easier to test and document